qiskit_experiments/hamiltonian_tomography/cr_hamiltonian_experiment.py

```diff
@@ -0,0 +1,299 @@
+    __analysis_class__ = CRHamiltonianAnalysis
+
+    def __init__(
+        self,
+        qubits: Tuple[int, int],
+        durations: np.ndarray,
+        cr_gate_schedule: Optional[Union[pulse.Schedule, pulse.ScheduleBlock]] = None,
+        x_values: Optional[np.ndarray] = None,
+    ):
+        r"""Create new CR Hamiltonian tomography experiment.
+
+        Args:
+            qubits: The qubit on which to run tomography.
+            durations: The pulse durations to scan. The maximum value should be sufficiently long
+                so that at least one cycle of oscillation of
+                the Bloch vector can be observed in the Bloch sphere of the target qubit.
+                Typically 100s ns to 10s μs depending on the drive power and
+                qubit-qubit coupling strength.
+            cr_gate_schedule: CR gate schedule to measure. Defaults to a single cross resonance
+                pulse with the flat-topped Gaussian envelope.
+            x_values: Net interaction time of the cross resonance gate. This value is used
+                to calculate Pauli coefficients of the CR Hamiltonian.
+                This defaults to the net duration of the single CR pulse.
+                The net CR pulse duration is determined by
+
+                .. math::
+
+                    \tau' = \tau - 2 {\rm (risefall)} \sigma + \sqrt{2\pi\sigma^2}
+
+                where :math:`\sigma` is the standard deviation of Gaussian edges and
+                `\tau` represents the total duration of the CR pulse.
+                If ``cr_gate_schedule`` is overridden without ``x_values``,
+                this will be identical to ``durations`` in units of second.
+
+        Raises:
+            QiskitError: When qubit number is not 2.
+            QiskitError: When ``cr_gate_schedule`` is specified without parameter named "duration".
+            QiskitError: When length of ``x_values`` and ``durations`` are different.
+        """
+        super().__init__(qubits)
+
+        if len(qubits) != 2:
+            raise QiskitError(
+                f"{self.__class__.__name__} is 2 qubit experiment. len({qubits}) != 2."
+            )
+
+        if cr_gate_schedule is not None and not cr_gate_schedule.get_parameters("duration"):
+            raise QiskitError(
+                "Target cross resonance schedule doesn't contain the parameter `duration`."
+            )
+
+        if x_values is not None and len(durations) != len(x_values):
+            raise QiskitError(
+                "Length of x values are not equivalent to durations; "
+                f"{len(self.experiment_options.durations)} != {len(x_values)}."
+            )
+
+        self._cr_gate_schedule = cr_gate_schedule
+        self._x_values = x_values
+
+        self.set_experiment_options(durations=durations)
+
+    @classmethod
+    def _default_experiment_options(cls) -> Options:
+        """Default option values used for CR Hamiltonian tomography."""
+        return Options(
```

Comment:
better to do `options = super()._default_experiment_options` and then add the experiment specific options.